_posts/2022-03-28-improving-readability-of-alias-commands.adoc

```diff
@@ -0,0 +1,207 @@
+42:be:48:4c:05:76:39:39:60:1c:d6:67:02:1d:00:ba:f6:96:a6:85:78:f7:df:de:e7:fa:67:c9:77:c7:85:ef:32:b2:33:ba:e5:80:c0:bc:d5:69:5d:02:82:01:00:16:a6:5c:58:20:48:50:70:4e:75:02:a3:97:57:04:0d:34:da:3a:34:78:c1:54:d4:e4:a5:c0:2d:24:2e:e0:4f:96:e6:1
+e:4b:d0:90:4a:bd:ac:8f:37:ee:b1:e0:9f:31:82:d2:3c:90:43:cb:64:2f:88:00:41:60:ed:f9:ca:09:b3:20:76:a7:9c:32:a6:27:f2:47:3e:91:87:9b:a2:c4:e7:44:bd:20:81:54:4c:b5:5b:80:2c:36:8d:1f:a8:3e:d4:89:e9:4e:0f:a0:68:8e:32:42:8a:5c:78:c4:78:c6:8d:05:27:b7
+:1c:9a:3a:bb:0b:0b:e1:2c:44:68:96:39:e7:d3:ce:74:db:10:1a:65:aa:2b:87:f6:4c:68:26:db:3e:c7:2f:4b:55:99:83:4b:b4:ed:b0:2f:7c:90:e9:a4:96:d3:a5:5d:53:5b:eb:fc:45:d4:f6:19:f6:3f:3d:ed:bb:87:39:25:c2:f2:24:e0:77:31:29:6d:a8:87:ec:1e:47:48:f8:7e:fb:
+5f:de:b7:54:84:31:6b:22:32:de:e5:53:dd:af:02:11:2b:0d:1f:02:da:30:97:32:24:fe:27:ae:da:8b:9d:4b:29:22:d9:ba:8b:e3:9e:d9:e1:03:a6:3c:52:81:0b:c6:88:b7:e2:ed:43:16:e1:ef:17:db:de:03:82:01:05:00:02:82:01:00:44:9a:19:f7:20:16:ab:f3:74:d9:17:93:b6:4
+8:d0:53:f6:e9:b7:b5:82:64:17:f9:de:e6:ab:5d:80:d7:ef:52:a1:b1:3e:f3:35:f2:31:7c:82:dd:57:78:30:b6:ae:2b:6c:ab:01:57:eb:2e:25:c3:69:2a:91:f8:67:fc:49:e2:31:7a:06:8f:ed:9f:58:d3:8a:75:3c:f2:9d:70:b1:4c:2f:3a:71:b4:e4:01:22:be:16:1a:f3:14:40:12:22
+:f1:82:94:da:fd:bf:b8:9c:c7:af:92:bc:24:ca:ae:32:03:9b:5f:8d:0e:5f:56:71:b4:39:6d:27:d0:68:19:dd:42:d4:d3:0f:61:76:39:f6:ad:f3:95:e5:30:0e:70:6e:ce:7c:aa:49:4d:24:4d:ad:4e:5f:a8:e7:42:bc:67:ba:8f:df:90:5f:6a:6c:f8:75:c5:31:b3:67:f9:b9:e7:6a:fa:
+44:35:85:d1:36:46:db:0b:51:58:6a:ae:f5:4d:ad:d8:65:8a:e1:68:04:34:bf:1b:d2:f2:60:f5:4f:33:df:c5:a1:7a:de:8e:54:7b:e5:20:29:eb:e9:40:11:15:07:aa:0c:f7:ba:69:00:60:98:44:73:e0:92:7f:a3:42:32:0c:a9:91:d6:7d:06:04:54:5c:21:dd:1d:f6:f4:96:3a:65:a3:2
+1:30:1f:30:1d:06:03:55:1d:0e:04:16:04:14:9c:85:1e:20:7f:82:fb:34:09:b2:8e:ad:30:9b:5d:60:e9:0c:13:2c:30:0b:06:09:60:86:48:01:65:03:04:03:02:03:3f:00:30:3c:02:1c:56:d1:21:21:47:29:76:15:88:c2:7f:d8:2e:0c:82:d3:fb:40:39:d8:f9:9a:2a:f7:87:1f:5b:ab
+:02:1c:54:66:42:6e:55:08:d5:4e:85:e3:e7:66:9f:8b:d8:ad:11:5e:63:91:60:9d:c2:a7:7b:ed:59:ce",
+            "subject" => "CN=www.example.com",
+            "issuer" => "CN=www.example.com",
+            "not-before" => "2022-03-28T10:21:44.000+0200",
+            "not-after" => "2022-06-26T10:21:44.000+0200",
+            "serial-number" => "08:f6:f0:9b:2c:35:23:9d",
+            "signature-algorithm" => "SHA256withDSA",
+            "signature" => "30:3c:02:1c:56:d1:21:21:47:29:76:15:88:c2:7f:d8:2e:0c:82:d3:fb:40:39:d8:f9:9a:2a:f7:87:1f:5b:ab:02:1c:54:66:42:6e:55:08:d5:4e:85:e3:e7:66:9f:8b:d8:ad:11:5e:63:91:60:9d:c2:a7:7b:ed:59:ce",
+            "version" => "v3"
+        }]
+    }
+}
+----
+
+Naturally, reducing verbosity of output, will make this command more user friendly, for instance:
```

Comment:
Maybe just to emphasize that this is now possible, this could be reworded to:

It is now possible to reduce the verbosity of the output, making this command more user friendly. For instance: